site_root/input/resources/examples/Task-se-001.json

```diff
@@ -0,0 +1,112 @@
+{
+  "resourceType": "Task",
+  "id": "se-001",
+  "meta": {
+    "profile": [
+      "http://fhir.kids-first.io/StructureDefinition/kfdrc-sequencing-experiment"
```

Comment:
I commented about it below (at the profile resource), but figured we neither add an extension(s) nor modify an existing attribute(s), given `Task`. I know we have created a new profile when a KF entity is a very specific use case off of a FHIR base profile, but without any addition, we may not create a new one. So, just `"http://hl7.org/fhir/StructureDefinition/Task"`?